joomla/joomla-cms
# [4.0] Extend removal of component sidebar

Pull Request for Issue # .

### Summary of Changes
This PR extends the removal of the component sidebar menu to all components where the component menu matches the main navigation sub menu (see #16386) .

Applies to the following...

- com_checkin
- com_cache
- com_cache&view=purge
- com_users&view=users
- com_users&view=groups
- com_users&view=levels
- com_users&view=notes
- com_menus&view=items
- com_banners
- com_banners&view=clients
- com_banners&view=tracks
- com_newsfeeds
- com_installer
- com_installer&view=update
- com_installer&view=manage
- com_installer&view=discover
- com_installer&view=database
- com_installer&view=warnings
- com_installer&view=languages
- com_installer&view=updatesites
- com_templates&view=styles
- com_templates&view=templates
- com_languages&view=installed
- com_languages&view=languages
- com_languages&view=overrides (Removed menu however still sidebar containing filter)


Excludes the following... 
- com_contact
- com_finder

Components excluded contain items in the component menu not present in the main menu. 

### Testing Instructions
Apply patch and ensure the sidebar is removed and table is displaying correctly (full width) at each of the following or navigate through all list views ensuring the component menu is removed where applicable (see #16386)....

/index.php?option=com_checkin
/index.php?option=com_cache
/index.php?option=com_cache&view=purge
/index.php?option=com_users&view=users
/index.php?option=com_users&view=groups
/index.php?option=com_users&view=levels
/index.php?option=com_users&view=notes
/index.php?option=com_menus&view=items
/index.php?option=com_banners
/index.php?option=com_banners&view=clients
/index.php?option=com_banners&view=tracks
/index.php?option=com_newsfeeds
/index.php?option=com_installer
/index.php?option=com_installer&view=update
/index.php?option=com_installer&view=manage
/index.php?option=com_installer&view=discover
/index.php?option=com_installer&view=database
/index.php?option=com_installer&view=warnings
/index.php?option=com_installer&view=languages
/index.php?option=com_installer&view=updatesites
/index.php?option=com_templates&view=styles
/index.php?option=com_templates&view=templates
/index.php?option=com_languages&view=installed
/index.php?option=com_languages&view=languages
/index.php?option=com_languages&view=overrides (Removed menu however still sidebar containing filter)

### Documentation Changes Required
Yes
